nltk/parse/stanford.py

```diff
@@ -127,23 +127,62 @@ def raw_parse(self, sentence, verbose=False):
         """
         return next(self.raw_parse_sents([sentence], verbose))
 
-    def raw_parse_sents(self, sentences, verbose=False):
-        """
-        Use StanfordParser to parse multiple sentences. Takes multiple sentences as a
-        list of strings.
-        Each sentence will be automatically tokenized and tagged by the Stanford Parser.
+    def raw_parse_sents(
+        self,
+        sentences,
+        verbose=False,
+        tokenize_whitespace=False,
+        sentence_split=False,
+    ):
+        """Use StanfordParser to parse multiple sentences.
 
-        :param sentences: Input sentences to parse
+        Takes multiple sentences as a list of strings. Each sentence will be
+        automatically tokenized and tagged by the Stanford Parser.
+
+        :param sentences: Input sentences to parse.
         :type sentences: list(str)
         :rtype: iter(iter(Tree))
+
         """
-        cmd = [
-            self._MAIN_CLASS,
-            '-model', self.model_path,
-            '-sentences', 'newline',
-            '-outputFormat', self._OUTPUT_FORMAT,
-        ]
-        return self._parse_trees_output(self._execute(cmd, '\n'.join(sentences), verbose))
+        session = requests.Session()
+
+        for sentence in sentences:
+
+            properties = {
```

Comment:
Possibly these properties should be set outside of the loop and I would personally prefer them to be set at `__init__()` so users can control the `annotators`, `outputFormat`, etc. before calling the `raw_parse_sents()`.